frontend/src/js/components/settings/organization/BillingDetails.tsx

```diff
@@ -58,7 +58,7 @@ export const BillingDetails = props => {
             {billing.address && <Address address={billing.address} email={billing.email} name={billing.name} />}
             {card && !hideCard && <CardDetails card={card} containerClass={billing.address ? 'margin-left-x-large' : ''} />}
           </div>
-          <Button className="margin-top-x-small" onClick={() => setChangeBilling(true)} size="small">
+          <Button disabled={editDisabled} variant="outlined" className="margin-top-x-small" onClick={() => setChangeBilling(true)} size="medium">
```

Comment:
```suggestion
          <Button disabled={editDisabled} variant="outlined" className="margin-top-x-small" onClick={() => setChangeBilling(true)}>
```
to reduce places to adjust if a new button size becomes the default 🙃